src/Core/StaticRandom.cs

```diff
@@ -36,29 +35,17 @@ public static void SeedWith(int seed)
         /// <summary>
         /// Returns a random integer.
         /// </summary>
-        public static int Int()
-        {
-            return Random.Next();
-        }
+        public static int Int() => Random.Next();
```

Comment:
Let's set up some property tests for these!

In a separate PR, that is. Will make an issue.